<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QMainWindow" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(47, 143, 140);
background-color: rgb(198, 174, 86);</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="passwd_label">
    <property name="geometry">
     <rect>
      <x>216</x>
      <y>167</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(32, 17, 255);</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="username_label">
    <property name="geometry">
     <rect>
      <x>213</x>
      <y>120</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(32, 17, 255);</string>
    </property>
    <property name="text">
     <string>User name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="username_le">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwd_le">
    <property name="geometry">
     <rect>
      <x>301</x>
      <y>160</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="user_label">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>110</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo_label">
    <property name="geometry">
     <rect>
      <x>301</x>
      <y>37</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:600; color:#41399d;&quot;&gt;LOGIN HERE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="veh_label">
    <property name="geometry">
     <rect>
      <x>139</x>
      <y>10</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>221</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QPushButton" name="btn1">
        <property name="toolTip">
         <string>font: bold;background-color: rgb(195, 193, 255);font-size: 36px;height: 48px;width: 120px</string>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(195, 193, 255);
font: 16pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn2">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(195, 193, 255);
font: 16pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn3">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(195, 193, 255);
font: 16pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QPushButton" name="btn4">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(195, 193, 255);
font: 16pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn5">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(195, 193, 255);
font: 16pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn6">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(195, 193, 255);
font: 16pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QSplitter" name="splitter_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QPushButton" name="btn7">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(195, 193, 255);
font: 16pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>7</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn8">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(195, 193, 255);
font: 16pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn9">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(195, 193, 255);
font: 16pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="cancel_btn">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(197, 158, 191);</string>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn0">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(195, 193, 255);
font: 16pt &quot;Sans Serif&quot;;
</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="login_btn">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(197, 158, 191);</string>
         </property>
         <property name="text">
          <string>Login</string>
         </property>
         <property name="iconSize">
          <size>
           <width>10</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>passwd_label</zorder>
   <zorder>username_label</zorder>
   <zorder>username_le</zorder>
   <zorder>passwd_le</zorder>
   <zorder>cancel_btn</zorder>
   <zorder>login_btn</zorder>
   <zorder>user_label</zorder>
   <zorder>logo_label</zorder>
   <zorder>btn0</zorder>
   <zorder>veh_label</zorder>
   <zorder>splitter_3</zorder>
   <zorder></zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
